tests/FakeItEasy.Specs/EventRaisingSpecs.cs

```diff
@@ -9,10 +9,13 @@ namespace FakeItEasy.Specs
     using Xbehave;
     using Xunit;
 
+    [SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Justification = "Best name to describe the type.")]

```

Comment:
[CA1711](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1711). When I switched everything to be just "Handler" instead of "EventHandler", `CustomEventHandler` came up with a [CA1710](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1710). I figured I could
1. annotate the 3 classes, 
2. mix the naming styles, or
3. change all and annotate CustomEventHandler

I picked this, but am happy to change.

Reply:
Whatever works! Again, I think I would have just disabled the warning in test code.

Reply:
Oh, I can change to disable globally within tests if you prefer.